frontend/src/routes/GlobalRoutes.tsx

```diff
@@ -1,25 +1,14 @@
-import React, {useEffect, useState} from 'react';
+import React from 'react';
 import Header from "../components/header/Header";
 import RouteMain from "./RouteMain";
 import RouteAuth from "./RouteAuth";
 import {useAppSelector} from "../stores/redux/hooks";
-import {selectLoggedUser} from "../stores/redux/loggedUser/loggedUserSlice";
-import _ from "lodash";
 import Notification from "../components/notification/Notification";
 import Footer from "../components/footer/Footer";
+import {isUserLogged} from "../stores/redux/loggedUser/loggedUserSlice";
 
 const GlobalRoutes: React.FC = () => {
-
-    const [isLogged, setIsLogged] = useState<boolean>(false);
-    const loggedUser = useAppSelector(selectLoggedUser);
-
-    const updateIsLogged = () => {
-        setIsLogged(!_.isEmpty(loggedUser.token));
-    }
-
-    useEffect(() => {
-        updateIsLogged();
-    }, [loggedUser]);
+    const isLogged = useAppSelector(isUserLogged);
```

Comment:
instead of renaming you just changed names.
isUserLogged is not what method does. It's name of the variable, but you have a method. You can name method same as you have in the example: `selectIsLogged`